train.py

```diff
@@ -312,24 +329,22 @@ def create_model(*_args, **kwargs):
 
         model.learn(n_timesteps, **kwargs)
 
-        # Save trained model
-        log_path = "{}/{}/".format(args.log_folder, args.algo)
-        save_path = os.path.join(log_path, "{}_{}".format(env_id, get_latest_run_id(log_path, env_id) + 1))
-        params_path = "{}/{}".format(save_path, env_id)
-        os.makedirs(params_path, exist_ok=True)
-
         # Only save worker of rank 0 when using mpi
         if rank == 0:
-            print("Saving to {}".format(save_path))
+            os.makedirs(save_path, exist_ok=True)
+            print("Saving model to {}".format(save_path))
+            model.save(save_path)
 
-            model.save("{}/{}".format(save_path, env_id))
             # Save hyperparams
-            with open(os.path.join(params_path, 'config.yml'), 'w') as f:
+            hyperparams_path = os.path.join(save_path, 'config.yml')
```

Comment:
it seems you changed the logic of where the model and where the parameters associated are saved